packages/curriculum-compiler-json/compilers/question.js

```diff
@@ -11,7 +11,8 @@ module.exports = function question(node, type) {
   const answers = getAnswersFromNode(node);
   const q = compileNodeToQuestionMarkdown({
     children: node.children.filter(
-      child => child.type !== 'list' && !child.answers
+      child =>
+        child.type !== 'heading' && child.type !== 'list' && !child.answers
```

Comment:
`heading` type introduced by markdown => question AST parser. Represents `## Revision` or `## Practice` in question text.

Reply:
Update: This is no longer necessary... `heading` node is now spliced from the children in the `compileQuestion` function.